lib/wsdl.js

```diff
@@ -1957,5 +1975,13 @@ function open_wsdl(uri, options, callback) {
   return wsdl;
 }
 
+function clear_cache() {
+  WSDL_CACHE = {};
```

Comment:
Okay, I guess I can see where it could make a difference.  If another reference had been created to WSDL_CACHE, it would be left with the old values if WSDL_CACHE was just reassigned.